### PR TITLE
FEAT: fix create_polyline for angular arc + update docs

### PR DESCRIPTION
**Pull request by gmalinve**

### CODE REVIEW

**Review by SMoraisAnsys — DISMISSED** [review]
LGTM, could you add a test to ensure that show the fix effect ?
This would avoid future regression and explain why tests were not failing despite that bug.

File: src/ansys/aedt/core/modeler/cad/polylines.py [hunk 1]
```diff
@@ -615,6 +615,7 @@ def _point_segment_string_array(self):
                     if current_segment.type == "AngularArc":
                         points_str.append(self._pl_point(current_segment.extra_points[i - 1]))
                         index_count += 1
+                        pos_count_incr += 1
```
Comment (SMoraisAnsys) [inline src/ansys/aedt/core/modeler/cad/polylines.py:618]: Can you add a test to show why previous implementation should have been failing?

Comment (gmalinve) [reply src/ansys/aedt/core/modeler/cad/polylines.py:618]: sure!

**Review by Samuelopez-ansys — APPROVED** [review]
Good job!

**Pull request merged**